bootstrap.sh

```diff
@@ -239,6 +239,13 @@ EOF
 
   echo "You may want to install the following:"
   sudo softwareupdate -l -a
+
+  git clone https://github.com/travis-ci/system-info.git
+  cd system-info
+  bundle install
+  bundle exec ruby exe/system-info
```

Comment:
I should have been more clear.  The above snippet suggestion assumes we want similar behavior on macOS as currently exists on linux.